Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM ghcr.io/ovrclk/akash:0.10.1
-LABEL org.opencontainers.image.source https://github.com/ovrclk/akash-on-akash
+FROM ghcr.io/ovrclk/akash:0.12.1
+# LABEL org.opencontainers.image.source https://github.com/ovrclk/akash-on-akash
```

Comment:
you can keep this, it links it back to the repo for the readme here: https://github.com/orgs/ovrclk/packages/container/package/akash-on-akash for instance